### PR TITLE
feat/bcrypt: add bcrypt command

### PR DESCRIPTION
**Pull request by zqwirp**
I've added the requested bcrypt command as explained in the issue. The `/bcrypt` command deletes the command and sends the hashed result of the command argument to the chat.

Please review the PR and let me know if anything needs to be added or adjusted.

### CODE REVIEW

**Comment by aldy505** [comment]
Is this still a draft, or ready to review?

**Comment by zqwirp** [comment]
> Is this still a draft, or ready to review?

ready for review

File: src/services/bcrypt/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+/**
+ *
+ * @param {import('telegraf').Context} context
+ * @returns {Promise<void>}
+ */
+async function bcrypt(context) {
+  const text = getCommandArgs("bcrypt", context);
+  const salt = bc.genSaltSync(10);
+  const hashedText = bc.hashSync(text, salt);
```
Comment (elianiva) [inline src/services/bcrypt/index.js:13]: are there any reasons behind the use of the sync version instead of its async counterpart?

Comment (zqwirp) [reply src/services/bcrypt/index.js:13]: just to write simple code

Comment (elianiva) [reply src/services/bcrypt/index.js:13]: I'm not sure if `bc.genSaltSync(10)` is any simpler than just doing `await bc.genSalt(10)`

Comment (zqwirp) [reply src/services/bcrypt/index.js:13]: got it, i'll update it to use `await bc.genSalt(10)`

**Review by aldy505 — APPROVED** [review]

**Review by lamualfa — APPROVED** [review]

**Comment by aldy505** [comment]
Hi @zqwirp! Your bcrypt implementation causes an error that makes the bot keeps restarting over and over again.

You can find the detail of the error here: https://teknologi-umum.sentry.io/share/issue/34a257c81f3c45a1b28b7b36aa1301b4/

On production, we're using Docker as defined on the Dockerfile (you can see it here https://github.com/teknologi-umum/bot/blob/master/Dockerfile). Any inputs are welcome.

**Comment by WahidinAji** [comment]
is it possible to revert this PR and let the existing command work as usual? ^-^

**Pull request merged**